databricks/koalas/base.py

```diff
@@ -579,8 +579,18 @@ def rmod(left, right):
 
         return column_op(rmod)(self, other)
 
-    __pow__ = column_op(Column.__pow__)
-    __rpow__ = column_op(Column.__rpow__)
+    def __pow__(self, other) -> Union["Series", "Index"]:
+        def pow_func(left, right):
+            return F.when(F.lit(right is np.nan) & (left == 1), left).otherwise(pow(left, right))
```

Comment:
I guess just checking `(left == 1)` is enough?

Otherwise:

```py
>>> ks.Series([1, 2, 3]) ** ks.Series([np.nan, 2, 3])
0     NaN
1     4.0
2    27.0
dtype: float64
>>> pd.Series([1, 2, 3]) ** pd.Series([np.nan, 2, 3])
0     1.0
1     4.0
2    27.0
dtype: float64
```